specification/metrics/listunitprice.md

```diff
@@ -1,8 +1,8 @@
 # List Unit Price
 
-The List Unit Price represents the suggested provider published unit price for a single pricing measurement unit, which incorporates volume/tier-based rates while excluding any negotiated or commitment-based discounts. This price is denominated in the [Billing Currency](#billingcurrency). It is used to calculate List Cost and savings based on rate optimization activities.
+The List Unit Price represents a suggested provider published unit price for a single pricing measurement unit, which incorporates volume/tier-based rates while excluding any negotiated or commitment-based discounts. This price is denominated in the [Billing Currency](#billingcurrency). It is used to calculate List Cost and savings based on rate optimization activities.
```

Comment:
Should be in sync with the [suggestion](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/235/files#r1370605836) :
...while excluding any negotiated, commitment-based discounts or spare capacity (spot) discounts...

Reply:
To include the capability to calculate savings from utilizing spare capacity (spot) discounts, we are still exploring a couple of alternative List Unit Price definitions.

- OPTION A (current version): _The List Unit Price represents a suggested provider-published unit price for a single pricing measurement unit (identified by a [Sku Price Id](#skupriceid)), which incorporates volume/tier-based rates while excluding any negotiated discounts, commitment-based discounts or the amortization of upfront charges (one-time or recurring)._

- OPTION B : _The List Unit Price represents a suggested provider-published unit price for a single pricing measurement unit (identified by a [Sku Price Id](#skupriceid)), which incorporates volume/tier-based rates while excluding any negotiated discounts, commitment-based discounts, **spare capacity (spot) discounts** or the amortization of upfront charges (one-time or recurring)._

- OPTION C (aligned with pricing model column terminology): _The List Unit Price represents a suggested provider-published unit price for a single pricing measurement unit (identified by a [Sku Price Id](#skupriceid)), which incorporates volume/tier-based rates while excluding any negotiated discounts, commitment-based discounts, **the impacts of dynamic pricing model** or the amortization of upfront charges (one-time or recurring)._

Note: **Alignment of all unit price and cost metric definitions is implied.**

Reply:
B is most clear but lacks flexibility of C.  Instead of "the impacts of dynamic pricing model" how about "dynamic pricing models (e.g. spare capacity (spot) discounts)"?

Reply:
I think we first need to determine if ListCost can/should behave in this way (excluding dynamic prices and replacing with 'fixed' prices of a similar (not equivalent) SKU. Once we have that decision (poll wording is in review), we should decide how to write this?

Reply:
Based on the further in-meeting discussion, a new version of the definition has been prepared. 
Closing this conversation and opening a related conversation available [here](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/234#discussion_r1382558297).